kolibri/core/assets/src/core_app_constructor.js

```diff
@@ -44,15 +48,22 @@ function Lib() {
  */
 module.exports = function CoreApp() {
   this.lib = new Lib();
+  this.resources = new ResourceManager(this);
   const mediator = new Mediator();
 
+  this.resources.registerResource('contentmetadata', 'id', ContentMetaDataResource);
```

Comment:
Would it be reasonable for all resources to automatically register themselves in their constructors?